### DISCUSSION TITLE
Not loading or displaying from prior session

### DISCUSSION THREAD

**Discussion by orlandoai23** [adieyal/sd-dynamic-prompts#304] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/127401105/224006794-91e890bd-e20a-4bcf-8e51-bfe10549cbd0.png)
![image](https://user-images.githubusercontent.com/127401105/224007056-e7f163dd-055a-4b82-9921-d3efea3440a6.png)
![image](https://user-images.githubusercontent.com/127401105/224007075-5340234e-6265-4aab-b655-2f392ed63645.png)

I was using dynamic prompts for the first time. Worked perfectly when I installed the extension. I then loaded a different prior session and it's not showing up and won't let me install (says it's already installed).

I'm sure it's user error, but I don't know how to correct.

**Comment by adieyal** [comment, marked as answer]
In your colab, try `! pip install --upgrade dynamicprompts`

Let me know if that works

**Reply by orlandoai23** [reply]
That worked, although there seemed to be an error within, but the end result was what I was seeking. Thank you! 
![image](https://user-images.githubusercontent.com/127401105/224070571-1df008bf-9533-48f5-8f31-d6de16c17fc9.png)
